pandas/tests/io/excel/test_readers.py

```diff
@@ -876,19 +876,11 @@ def test_corrupt_bytes_raises(self, engine):
             pd.read_excel(bad_stream)
 
     @pytest.mark.network
```

Comment:
Similarly, do these need to be marked network anymore? I suppose we're still going over the network, even if it's an internal server, and that could pose issues for different setups?

Reply:
I decided to keep making these as `network` just for consistency, even as you mentioned its a local network